benchexec/tools/nitwit.py

```diff
@@ -50,24 +43,24 @@ def name(self):
     def cmdline(self, executable, options, task, rlimits):
         if task.property_file:
             options = options + ["-p", task.property_file]
-        return [executable] + options + list(task.input_files_or_identifier)
+        return [executable] + options + task.single_input_file
```

Comment:
```suggestion
        return [executable] + options + [task.single_input_file]
```
Otherwise there is an error.

I suggest to always test the tool-info module with `python3 -m benchexec.test_tool_info nitwit --tool-directory ...`.